src/instruments/src/EWD/elements/PseudoFWC.tsx

```diff
@@ -0,0 +1,1597 @@
+            memoInhibit: false,
+            failure: 0,
+            sysPage: -1,
+            side: 'RIGHT',
+        },
+        '0000200': // PARK BRK
+        {
+            flightPhaseInhib: [3, 4, 5, 6, 7, 8],
+            simVarIsActive: !!([1, 2, 9, 10].includes(flightPhase) && parkBrake === 1),
+            whichCodeToReturn: [0],
+            codesToReturn: ['000020001'],
+            memoInhibit: false,
+            failure: 0,
+            sysPage: -1,
+            side: 'RIGHT',
+        },
+        '0000040': // NW STRG DISC
+        {
+            flightPhaseInhib: [],
+            simVarIsActive: nwSteeringDisc === 1,
+            whichCodeToReturn: [engine1State > 0 || engine2State > 1 ? 1 : 0],
+            codesToReturn: ['000004001', '000004002'],
+            memoInhibit: false,
+            failure: 0,
+            sysPage: -1,
+            side: 'RIGHT',
+        },
+        '0000160': // PTU ON
+        {
+            flightPhaseInhib: [],
+            simVarIsActive: hydPTU === 1,
+            whichCodeToReturn: [0],
+            codesToReturn: ['000016001'],
+            memoInhibit: false,
+            failure: 0,
+            sysPage: -1,
+            side: 'RIGHT',
+        },
+        '0000210': // RAT OUT
+        {
+            flightPhaseInhib: [],
+            simVarIsActive: ratDeployed > 0,
+            whichCodeToReturn: [[1, 2].includes(flightPhase) ? 1 : 0],
+            codesToReturn: ['000021001', '000021002'],
+            memoInhibit: false,
+            failure: 0,
+            sysPage: -1,
+            side: 'RIGHT',
+        },
+        '0000070': // IGNITION
+        {
+            flightPhaseInhib: [],
+            simVarIsActive: engSelectorPosition === 2,
+            whichCodeToReturn: [0],
+            codesToReturn: ['000007001'],
+            memoInhibit: false,
+            failure: 0,
+            sysPage: -1,
+            side: 'RIGHT',
+        },
+        '0000540': // PRED W/S OFF
+        {
+            flightPhaseInhib: [1, 10],
+            simVarIsActive: !!(predWSOn === 0 && ![1, 10].includes(flightPhase)),
+            whichCodeToReturn: [[3, 4, 5, 7, 8, 9].includes(flightPhase) || toconfig === 1 ? 1 : 0],
+            codesToReturn: ['000054001', '000054002'],
+            memoInhibit: false,
+            failure: 0,
+            sysPage: -1,
+            side: 'RIGHT',
+        },
+        '0000545': // TERR OFF
+        {
+            flightPhaseInhib: [1, 10],
+            simVarIsActive: !!(gpwsOff === 1 && ![1, 10].includes(flightPhase)),
+            whichCodeToReturn: [[3, 4, 5, 7, 8, 9].includes(flightPhase) || toconfig === 1 ? 1 : 0],
+            codesToReturn: ['000054501', '000054502'],
+            memoInhibit: false,
+            failure: 0,
+            sysPage: -1,
+            side: 'RIGHT',
+        },
+        '0000320': // TCAS STBY
+        {
+            flightPhaseInhib: [],
+            simVarIsActive: !!(tcasMode === 0 && flightPhase !== 6),
```

Comment:
If you want to be technically more correct:

```suggestion
            simVarIsActive: !!(tcasSensitity == 1 && flightPhase !== 6),
```

where

``` 
const [tcasSensitivity] = useSimVar('L:A32NX_TCAS_SENSITIVITY', 'enum', 500);
```

...but this would need testing, not sure if TCAS does this correctly already.

[Reference](https://manualzz.com/doc/22316884/arinc-specification-429p2-16):
![image](https://user-images.githubusercontent.com/929769/156945505-a4e9479f-aebd-4e20-8474-1eac48415271.png)